src/xunit.performance.execution/BenchmarkTestInvoker.cs

```diff
@@ -252,7 +252,7 @@ protected override void StopMeasurement(int iterationNumber)
                     Debug.Assert(_currentIterationMeasurementStarted);
                     _currentIterationMesaurementStopped = true;
 
-                    BenchmarkEventSource.Log.BenchmarkIterationStop(BenchmarkConfiguration.Instance.RunId, _testName, iterationNumber);
+                    BenchmarkEventSource.Log.BenchmarkIterationStop(BenchmarkConfiguration.Instance.RunId, _testName, iterationNumber, 0);
```

Comment:
You should calculate allocated bytes here before calling BenchmarkIterationStop and then pass it in here.  Then everything works out with traceparsergen and you can do whatever calculation you need outside of the eventsource (it's really just for logging).